lib/board.rb

```diff
@@ -0,0 +1,62 @@
+class Board
+
+  attr_accessor :cells #has an attribute to store the cells of the board
+
+  def initialize #sets the cells of the board to a 9 element array of " "
+    @cells = Array.new(9," ")
```

Comment:
I would just call `self.reset!` here and delegate to that method to create the default empty board.